pal/pal_pc/src/led.rs

```diff
@@ -1,10 +1,15 @@
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub enum Led {
-    Green,
-    Red,
+#[derive(Debug, PartialEq, Eq)]
+pub enum LedState {
+    Off,
+    TokenNotProvisioned,
+    TokenProvisioningComplete,
+    TokenWaiting,
+    PlatformProvisioningOk,
+    AttestationOk,
+    AttestationFailed,
 }
 
-pub fn control(_led: Led, _on: bool) {
+pub fn control(_state: LedState) {
```

Comment:
I think we should copy `trace!("LED controller state: {:?}", state);` from nRF PAL here to have at least some way of showing what LEDs would do. Maybe even bump verbosity level for PC target, real LEDs are always visible after all.